Fail to deploy to k3s cluster

Hi! Thanks for the effort creating this chart. I was thinking about the same before stumbled upon your work.
I am trying to deploy it to k3s cluster via its custom [Helm resource definition](https://rancher.com/docs/k3s/latest/en/helm) like this:

```yaml
apiVersion: helm.cattle.io/v1
kind: HelmChart
metadata:
  name: wg-easy
  namespace: kube-system
spec:
  chart: wg-easy
  repo: "https://brandon099.github.io/wg-easy-helm-chart"
  targetNamespace: kube-system
  version: "0.1.8"
``` 
But the helm-install-wg-easy pod fails to install it with the following error:
```
CHART=$(sed -e "s/%{KUBERNETES_API}%/${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}/g" <<< "${CHART}")
set +v -x
+ cp /var/run/secrets/kubernetes.io/serviceaccount/ca.crt /usr/local/share/ca-certificates/
+ update-ca-certificates
WARNING: ca-certificates.crt does not contain exactly one certificate or CRL: skipping
+ '[' '' '!=' true ']'
+ export HELM_HOST=127.0.0.1:44134
+ HELM_HOST=127.0.0.1:44134
+ tiller --listen=127.0.0.1:44134 --storage=secret
+ helm_v2 init --skip-refresh --client-only --stable-repo-url https://charts.helm.sh/stable/
Creating /root/.helm 
[main] 2021/12/11 21:01:47 Starting Tiller v2.16.10 (tls=false)
[main] 2021/12/11 21:01:47 GRPC listening on 127.0.0.1:44134
[main] 2021/12/11 21:01:47 Probes listening on :44135
[main] 2021/12/11 21:01:47 Storage driver is Secret
[main] 2021/12/11 21:01:47 Max history per release is 0
Creating /root/.helm/repository 
Creating /root/.helm/repository/cache 
Creating /root/.helm/repository/local 
Creating /root/.helm/plugins 
Creating /root/.helm/starters 
Creating /root/.helm/cache/archive 
Creating /root/.helm/repository/repositories.yaml 
Adding stable repo with URL: https://charts.helm.sh/stable/ 
Adding local repo with URL: http://127.0.0.1:8879/charts 
$HELM_HOME has been configured at /root/.helm.
Not installing Tiller due to 'client-only' flag having been set
++ helm_v2 ls --all '^wg-easy$' --output json
++ jq -r '.Releases | length'
[storage] 2021/12/11 21:01:47 listing all releases with filter
+ EXIST=
+ '[' '' == 1 ']'
+ '[' '' == v2 ']'
+ shopt -s nullglob
+ helm_content_decode
+ set -e
+ ENC_CHART_PATH=/chart/wg-easy.tgz.base64
+ CHART_PATH=/wg-easy.tgz
+ '[' '!' -f /chart/wg-easy.tgz.base64 ']'
+ return
+ '[' install '!=' delete ']'
+ helm_repo_init
+ grep -q -e 'https\?://'
+ '[' helm_v3 == helm_v3 ']'
+ [[ wg-easy == stable/* ]]
+ '[' -n https://brandon099.github.io/wg-easy-helm-chart ']'
+ helm_v3 repo add wg-easy https://brandon099.github.io/wg-easy-helm-chart
"wg-easy" has been added to your repositories
+ helm_v3 repo update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "wg-easy" chart repository
Update Complete. ⎈Happy Helming!⎈
+ helm_update install --namespace kube-system --repo https://brandon099.github.io/wg-easy-helm-chart --version 0.1.8
+ '[' helm_v3 == helm_v3 ']'
++ helm_v3 ls -f '^wg-easy$' --namespace kube-system --output json
++ jq -r '"\(.[0].app_version),\(.[0].status)"'
++ tr '[:upper:]' '[:lower:]'
+ LINE=null,null
++ echo null,null
++ cut -f1 -d,
+ INSTALLED_VERSION=null
++ echo null,null
++ cut -f2 -d,
+ STATUS=null
+ VALUES=
+ '[' install = delete ']'
+ '[' -z null ']'
+ '[' null = deployed ']'
+ '[' null = failed ']'
+ '[' null = deleted ']'
+ helm_v3 install --namespace kube-system --repo https://brandon099.github.io/wg-easy-helm-chart --version 0.1.8 wg-easy wg-easy
Error: YAML parse error on wg-easy/templates/deployment.yaml: error converting YAML to JSON: yaml: line 73: did not find expected '-' indicator
```

I've forked this project for further investigation of the issue, but maybe you've already faced it and know what is the problem? Will be appreciated for any help.

I've run `helm lint charts/wg-easy` and here is the error:

```
==> Linting .
[ERROR] Chart.yaml: appVersion should be of type string but it's of type float64
[INFO] Chart.yaml: icon is recommended
[ERROR] templates/deployment.yaml: unable to parse YAML: error converting YAML to JSON: yaml: line 73: did not find expected '-' indicator
```
Gonna try to fix it

--- a/charts/wg-easy/Chart.yaml
+++ b/charts/wg-easy/Chart.yaml
@@ -4,6 +4,6 @@ description: A wg-easy Wireguard server
 type: application
 icon: https://www.wireguard.com/img/wireguard.svg
 # This is the chart version.
-version: 0.1.9
+version: 0.1.10
 # This is the version number of the application being deployed.
 appVersion: 4

--- a/charts/wg-easy/templates/deployment.yaml
+++ b/charts/wg-easy/templates/deployment.yaml
@@ -93,7 +93,7 @@ spec:
             claimName: {{ if .Values.persistence.existingClaim }}{{ .Values.persistence.existingClaim }}{{- else }}{{ include "wg-easy.fullname" . }}{{- end }}
         {{- end }}
         {{- if not .Values.persistence.enabled }}
-        emptyDir: {}
+          emptyDir: {}
         {{- end }}
       {{- with .Values.nodeSelector }}
       nodeSelector:

--- a/charts/wg-easy/values.yaml
+++ b/charts/wg-easy/values.yaml
@@ -16,14 +16,12 @@ web:
   service:
     type: ClusterIP
     port: 51821
-    externalTrafficPolicy: Local
     annotations: {}
 
 wireguard:
   service:
     type: ClusterIP
     port: 51820
-    externalTrafficPolicy: Local
     annotations: {}
 
   # Wireguard host is required and used when creating client config

